run/run.py

```diff
@@ -218,7 +215,7 @@ def main(args):
         '--log-raw', LOG_FILEPATH,
         '--log-wptreport', REPORT_FILEPATH,
         '--log-mach=-',
-        '--processes=24', # FIXME remove insane mode option
+        '--processes=2', # FIXME remove insane mode option
```

Comment:
Do we know how this affects stability? I have always assumed that using processes > 1 in production would increase the rate of intermittency.